Formula/observerward.rb

```diff
@@ -0,0 +1,18 @@
+class Observerward < Formula
+  desc "Cross platform community web fingerprint identification tool"
+  homepage "https://0x727.github.io/ObserverWard/"
+  url "https://github.com/0x727/ObserverWard/archive/refs/tags/v2022.3.15.tar.gz"
+  sha256 "e1420fe744edbafaaf267fb85e3e016c3cca454117e1ffd360d6d25debf2f560"
+  license "MIT"
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    system "#{bin}/observer_ward", "-u"
```

Comment:
```suggestion
    system bin/"observer_ward", "-u"
```
You don't need interpolation here.

Reply:
I have made modifications, please check